include/modmesh/python/common.hpp

```diff
@@ -300,27 +306,27 @@ WrapBase
         static_assert(!std::is_const<array_reference>::value, "this_array_reference cannot be const");
         using array_type = typename std::remove_reference<array_reference>::type;
 
-        (*this).def_property
-        (
-            name
-          , [&f](wrapped_type & self) -> array_reference { return f(self); }
-          , [&f](wrapped_type & self, py::array_t<typename array_type::value_type> & ndarr)
-            {
-                array_reference this_array = f(self);
-                if (this_array.nbytes() != static_cast<size_t>(ndarr.nbytes()))
+        (*this)
+            .def_property(
+                name,
+                [&f](wrapped_type & self) -> array_reference
+                { return f(self); },
+                [&f](wrapped_type & self, py::array_t<typename array_type::value_type> & ndarr)
                 {
-                    std::ostringstream msg;
-                    msg
-                        << ndarr.nbytes() << " bytes of input array differ from "
-                        << this_array.nbytes() << " bytes of internal array"
-                    ;
-                    throw std::length_error(msg.str());
-                }
-                makeSimpleArray(ndarr).swap(this_array);
-            }
-        );
-
-        return *static_cast<wrapper_type*>(this);
+                    array_reference this_array = f(self);
+                    if (this_array.nbytes() != static_cast<size_t>(ndarr.nbytes()))
+                    {
+                        std::ostringstream msg;
+                        msg << ndarr.nbytes() << " bytes of input array differ from "
+                            << this_array.nbytes() << " bytes of internal array";
+                        throw std::length_error(msg.str());
+                    }
+                    makeSimpleArray(ndarr).swap(this_array);
+                })
+            //
```

Comment:
This is needed or the last semi-colon will be placed after the closing parenthsis.  Copying, pasting, and swapping the wrapping code is easy when the ending semi-colon use a standalone line:

```cpp
(*this)
    // Swapping A and B is easy.
    .def_property_readonly("A", &wrapped_type::A)
    .def_property_readonly("B", &wrapped_type::B)
    // The semi-colon is in a standalone line.
    ;
```

When the semi-colon is at the end of each of the wrapping line, swapping is hard:

```cpp
(*this)
    .def_property_readonly("A", &wrapped_type::A)
    .def_property_readonly("B", &wrapped_type::B); // The semi-colon makes it hard to swap with A.
```